src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -971,6 +991,9 @@ QueryNetworkResult Engine::QueryNetwork(const CNNNetwork& network, const std::ma
             }
         }
 
+        auto removedNodeNames = GetRemovedNodes(function, clonnedFunction);
```

Comment:
It should be gotten from fused names vector. If it is missed then the problem is in one of the transformations that missed to add it.

Reply:
We have an issue that not all transformation saved fused names in this rt section.